dht.go

```diff
@@ -181,6 +188,10 @@ func New(ctx context.Context, h host.Host, options ...Option) (*IpfsDHT, error)
 	// go-routine to make sure we ALWAYS have RT peer addresses in the peerstore
 	// since RT membership is decoupled from connectivity
 	go dht.persistRTPeersInPeerStore()
+
+	// listens to the fix low peers chan and tries to fix the Routing Table
+	go dht.fixLowPeersRoutine()
```

Comment:
later: given that we're using goprocess, let's manage this with goprocess.

Reply:
Done.